vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/DatabaseWorkflow.java

```diff
@@ -578,7 +578,7 @@ public void succeeded(DSLContext transaction) {
     transaction.deleteFrom(ACTIVE_WORKFLOW_RUN).where(ACTIVE_WORKFLOW_RUN.ID.eq(id)).execute();
     target
         .consumableResources()
-        .forEach(cr -> cr.second().release(workflowName, workflowVersion, vidarrId));
+        .forEach(cr -> cr.second().release(workflowName, workflowVersion, vidarrId, Optional.empty()));
```

Comment:
Similarly, this call happens when a workflow run becomes Succeeded, it will also get put into the WaitingState at low priority

Reply:
Good point, would it make more sense for release to do nothing? Behaving more like the `AlertmanagerAutoInhibitConsumableResource` ?

Reply:
Probably! Which would mean you don't need this extra parameter in the interface too

Reply:
I'm ok with reverting that change. I just want to check with @callunity since the test was deliberately configured to ensure that the workflow run WAS added back to waiting

Reply:
Looking at the various places that `release` is called, I agree it makes most sense for  PriorityByWorkflow's `release` to do nothing.

Reply:
(the test was to ensure it got re-added in case another consumable resource failed its check, but that's not necessary to re-add it because priority will get reassessed the next time the workflow run's consumable resources get checked)